webhooks/tasks/automaticTeams.js

```diff
@@ -35,6 +36,11 @@ module.exports = {
     const eventType = data.properties.event;
     const eventAction = data.body.action;
 
+    // Someone added a team to the repo
+    if (eventType === 'team' && (eventAction === 'add_repository' || eventAction === 'added_to_repository')) {
```

Comment:
I would usually use something like: `['add_repository', 'added_to_repository'].includes(eventAction)`, not sure if it makes the code clearer.